ts/session/utils/MessageUtils.ts

```diff
@@ -0,0 +1,89 @@
+import { RawMessage } from '../types/RawMessage';
+import { ContentMessage } from '../messages/outgoing';
+import { EncryptionType } from '../types/EncryptionType';
+import * as crypto from 'crypto';
+
+function toRawMessage(device: PubKey, message: ContentMessage): RawMessage {
+  const ttl = message.ttl();
+  const timestamp = message.timestamp;
+  const plainTextBuffer = message.plainTextBuffer();
+
+  // Get EncryptionType depending on message type.
+  // let encryption: EncryptionType;
+
+  // switch (message.constructor.name) {
```

Comment:
you can just use `message instanceof ContentMessage` etc..
All messages should default to `Signal` encryption. Only `SessionReset` messages (`message instanceof SessionResetMessage`) should be `SessionReset`